include/arc_utilities/arc_helpers.hpp

```diff
@@ -505,99 +480,37 @@ namespace arc_helpers
-            }
-            else if (color_code == 5)
-            {
-                return RGBAColorBuilder<ColorType>::MakeFromMixedColors(0xc1, 0x20, 0x00, alpha);
-            }
-            else if (color_code == 6)
-            {
-                return RGBAColorBuilder<ColorType>::MakeFromMixedColors(0xce, 0x62, 0xa2, alpha);
-            }
-            else if (color_code == 7)
-            {
-                return RGBAColorBuilder<ColorType>::MakeFromMixedColors(0x81, 0x66, 0x70, alpha);
-            }
-            else if (color_code == 8)
-            {
-                return RGBAColorBuilder<ColorType>::MakeFromMixedColors(0x00, 0x34, 0x7d, alpha);
-            }
-            else if (color_code == 9)
-            {
-                return RGBAColorBuilder<ColorType>::MakeFromMixedColors(0xf6, 0x8e, 0x76, alpha);
-            }
-            else if (color_code == 10)
-            {
-                return RGBAColorBuilder<ColorType>::MakeFromMixedColors(0x00, 0x8a, 0x53, alpha);
-            }
-            else if (color_code == 11)
-            {
-                return RGBAColorBuilder<ColorType>::MakeFromMixedColors(0xff, 0x5c, 0x7a, alpha);
-            }
-            else if (color_code == 12)
-            {
-                return RGBAColorBuilder<ColorType>::MakeFromMixedColors(0x53, 0x7a, 0x37, alpha);
-            }
-            else if (color_code == 13)
-            {
-                return RGBAColorBuilder<ColorType>::MakeFromMixedColors(0xff, 0x00, 0x8e, alpha);
-            }
-            else if (color_code == 14)
-            {
-                return RGBAColorBuilder<ColorType>::MakeFromMixedColors(0xb3, 0x51, 0x28, alpha);
-            }
-            else if (color_code == 15)
-            {
-                return RGBAColorBuilder<ColorType>::MakeFromMixedColors(0xf4, 0x00, 0xc8, alpha);
-            }
-            else if (color_code == 16)
-            {
-                return RGBAColorBuilder<ColorType>::MakeFromMixedColors(0x7f, 0x0d, 0x18, alpha);
-            }
-            else if (color_code == 17)
-            {
-                return RGBAColorBuilder<ColorType>::MakeFromMixedColors(0x93, 0x00, 0xaa, alpha);
-            }
-            else if (color_code == 18)
-            {
-                return RGBAColorBuilder<ColorType>::MakeFromMixedColors(0x59, 0x15, 0x33, alpha);
-            }
-            else if (color_code == 19)
-            {
-                return RGBAColorBuilder<ColorType>::MakeFromMixedColors(0xf1, 0x13, 0x3a, alpha);
-            }
-            else
-            {
-                return RGBAColorBuilder<ColorType>::MakeFromMixedColors(0x23, 0x16, 0x2c, alpha);
-            }
-        }
-        else
-        {
-            return RGBAColorBuilder<ColorType>::MakeFromFloatColors(0.0f, 0.0f, 0.0f, alpha);
-        }
+        std::map<uint32_t, std::vector<uint8_t>> color_map{
+            {0, {0xff, 0xff, 0xff}},
```

Comment:
This changse the logic, the old code has alpha=0 for this case. `sdf_tools` relies on this behaviour for better or worse.